src/Maths/math.vector.ts

```diff
@@ -355,14 +355,16 @@ export class Vector2 {
 
     /**
      * Gets a new Vector2 from current Vector2 floored values
+     * eg (1.2, 2.31, 1.54) returns (1, 2, 1)

```

Comment:
if you are already adding a comment like that to vector2, i think it will be better to have the example contain only 2 values and not 3.

Reply:
Yes, makes sense